### PR TITLE
Feature/adrs

### PR DESCRIPTION
**Pull request by st3v3nhunt**

### CODE REVIEW

File: doc/adr/0002-use-referer-header-for-id-acquisition.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Date: 2017-06-21
+
+## Status
+
+Accepted
+
+## Context
+
+The mechanism for directing people to the new GP Profile pages is to use an
+existing system where a banner is displayed on the current page. From this
```
Comment (beseven) [inline doc/adr/0002-use-referer-header-for-id-acquisition.md:12]: Are you going to reference Choices here? An example url?

File: doc/adr/0002-use-referer-header-for-id-acquisition.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+We will use the `referer` header to identify the ID of the GP profile page the
+user was previously viewing in order to redirect them to the new page.
+
+## Consequences
+
+If the `referer` is unavailable the ID of the GP will not be able to be
+determined. Situations where no `referer` is available include:
+* Browsers configured to block the transmission of `referer` headers
+* Some versions of
+  [IE](https://support.microsoft.com/en-us/help/178066/info-internet-explorer-does-not-send-referer-header-in-unsecured-situations)
```
Comment (beseven) [inline doc/adr/0002-use-referer-header-for-id-acquisition.md]: Do you have to explain that the consequence of this is a 404 page for the user?

**Review by beseven — CHANGES_REQUESTED** [review]
Some comments below

File: doc/adr/0002-use-referer-header-for-id-acquisition.md (hunk 2 above)
Comment (st3v3nhunt) [reply doc/adr/0002-use-referer-header-for-id-acquisition.md]: Yes. Update incoming...

File: doc/adr/0002-use-referer-header-for-id-acquisition.md (hunk 1 above)
Comment (st3v3nhunt) [reply doc/adr/0002-use-referer-header-for-id-acquisition.md:12]: No, the banners are temporary. I feel there is enough information provided to understand the decision around using the header.

**Comment by st3v3nhunt** [comment]
@beseven Updated with the rest of the consequence!

**Review by beseven — APPROVED** [review]

**Pull request merged**